sdk/keyvault/azure-keyvault-keys/azure/keyvault/keys/crypto/_client.py

```diff
@@ -211,6 +212,15 @@ def _initialize(self, **kwargs) -> None:
             # try to get the key again next time unless we know we're forbidden to do so
             self._initialized = self._keys_get_forbidden
 
+    @distributed_trace
+    def create_rsa_private_key(self) -> KeyVaultRSAPrivateKey:
```

Comment:
Were you still going to add public key support like we have in some other clients?

Reply:
Yep! Working on that now.